readme.md

```diff
@@ -113,7 +113,7 @@ AVA supports the latest release of any major version that [is supported by Node.
 
 ### Magic assert
 
-AVA adds code excerpts and clean diffs for actual and expected values. If values in the assertion are objects or arrays, only a diff is displayed, to remove the noise and focus on the problem. The diff is syntax-highlighted too! If you are comparing strings, both single and multi line, AVA displays a different kind of output, highlighting the added or missing characters.
+AVA adds code excerpts and clean diffs for actual and expected values. If values in the assertion are objects or arrays, only a diff is displayed, to remove the noise and focus on the problem. The diff is syntax-highlighted too! If you are comparing strings, both single and multi lines, AVA display a different kind of output, highlighting the added or missing characters.
```

Comment:
`multi line` here refers to `strings`, if you read it as `multi line strings` the original is correct.

The subject of `displays` is AVA, which is singular, so `displays` is correct.